src/bindgen/ir/ty.rs

```diff
@@ -282,7 +282,19 @@ impl Type {
                 Type::Array(Box::new(converted), len)
             }
             &syn::Type::BareFn(ref function) => {
-                let args = function.inputs.iter().try_skip_map(|x| Type::load(&x.ty))?;
+                let args = function.inputs.iter().try_skip_map(|x| {
+                    Type::load(&x.ty).map(|opt_ty| {
+                        opt_ty.map(|ty| {
+                            (
+                                x.name.as_ref().map(|name| match name.0 {
+                                    syn::BareFnArgName::Named(ref ident) => ident.to_string(),
+                                    syn::BareFnArgName::Wild(_) => "_".to_owned(),
```

Comment:
This will generate duplicate names when parsing a function pointer with multiple wild function arg names. We should add a monotonic counter to this. e.g. `_1, _2, _3, _4`.